CMakeLists.txt

```diff
@@ -268,16 +279,16 @@ endif()
 set(AER_LIBRARIES
 	${AER_LIBRARIES}
 	${BLAS_LIBRARIES}
-	CONAN_PKG::nlohmann_json
+	AER_DEPENDENCY_PKG::nlohmann_json
+	AER_DEPENDENCY_PKG::spdlog
 	Threads::Threads
-	CONAN_PKG::spdlog
 	${DL_LIB}
-	${THRUST_DEPENDANT_LIBS})
+	${THRUST_DEPENDENT_LIBS})
 
 set(AER_COMPILER_DEFINITIONS ${AER_COMPILER_DEFINITIONS} ${CONAN_DEFINES})
 # Cython build is only enabled if building through scikit-build.
 if(SKBUILD) # Terra Addon build
-	set(AER_LIBRARIES ${AER_LIBRARIES} CONAN_PKG::muparserx)
+	set(AER_LIBRARIES ${AER_LIBRARIES} AER_DEPENDENCY_PKG::muparserx)
```

Comment:
This is probably on me, but I'm wondering if we should be using `list()` here instead of `set()` , we are using `list()` in other places for appending libraries (see `AER_CONAN_LIBS` usage), also feels more semantically correct. Thoughts?

Reply:
BTW, I think I started to use `set()` instead of `list()` due to some misbehavior in CMake, I just can't recall exactly which one. If  `list()` works and covers our use-cases then go to `list()`. Maybe in another PR with a "Good Fist Issue" tag?.

Reply:
yeah, i think `list` is the way to go. We can check that it works and and replace all those `set`s.

Reply:
yeah, i think list is the way to go. We can check that it works and and replace all those sets.